omero/developers/Web/CreateApp.txt

```diff
@@ -358,6 +358,40 @@ of links at the top of the webclient main pages.
        $ bin/omero config append omero.web.ui.top_links '["Homepage", "http://myhome.com", {"title": "Homepage", "target": "new"}]'
 
 
+Custom image viewer
+^^^^^^^^^^^^^^^^^^^
+
+If you have created your own image viewer and would like to have it replace the existing image viewer in the
+webclient, this can be configured using :property:'omeroweb.webclient.views.image_viewer'.
+
+You will need your views.py method to take an Image ID with a parameter named ``iid``. For example, see
+``channel_overlay_viewer`` from the `webtest <https://github.com/openmicroscopy/webtest/>`_ app:
+
+   ::
```

Comment:
If you put the `::` at the end of the sentence and just indent the code snippets by 4 spaces, you'll get rid of the purple highlighting in plone that JM has pointed out.